hoomd/md/constrain.py

```diff
@@ -273,49 +130,43 @@ class rigid(ConstraintForce):
     orientation quaternion defines the rotation from the body space into the
     simulation box. In body space, the center of mass of the body is at 0,0,0
     and the moment of inertia is diagonal. You specify the constituent particles
-    to :py:class:`rigid` for each type of body in body coordinates. Then,
-    :py:class:`rigid` takes control of those particles, and sets their position
+    to `Rigid` for each type of body in body coordinates. Then,
+    `Rigid` takes control of those particles, and sets their position
     and orientation in the simulation box relative to the position and
-    orientation of the central particle. :py:class:`rigid` also transfers forces
+    orientation of the central particle. `Rigid` also transfers forces
     and torques from constituent particles to the central particle. Then, MD
     integrators can use these forces and torques to integrate the equations of
     motion of the central particles (representing the whole rigid body) forward
     in time.
 
     .. rubric:: Defining bodies
 
-    :py:class:`rigid` accepts one local body environment per body type. The
+    `Rigid` accepts one local body definition per body type. The
     type of a body is the particle type of the central particle in that body.
     In this way, each particle of type *R* in the system configuration defines
     a body of type *R*.
 
     As a convenience, you do not need to create placeholder entries for all of
     the constituent particles in your initial configuration. You only need to
     specify the positions and orientations of all the central particles. When
-    you call :py:meth:`create_bodies()`, it will create all constituent
-    particles that do not exist. (those that already exist e.g. in a restart
-    file are left unchanged).
+    you call `~.create_bodies`, it will create all constituent particles.
```

Comment:
```suggestion
    you call `create_bodies`, it will create all constituent particles.
```
I think this will link correctly now.